### PR TITLE
New package: eternity-4.02.00

### PR DESCRIPTION
**Pull request by Antkibo**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/eternity/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+And check if the following directories exist:
+
+	* doom
+	* doom2
+	* hacx
+	* heretic
+	* plutonia
+	* shots
+	* tnt
```
Comment (ericonr) [inline srcpkgs/eternity/INSTALL.msg:13]: `make_dirs` should be used in the template instead, though the software needing these directories to exist doesn't fill me with confidence.

File: srcpkgs/eternity/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#archs="i686 x86_64"
+#wrksrc=
+create_wrksrc=yes
+build_wrksrc="${pkgname}-${version}"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
```
Comment (ericonr) [inline srcpkgs/eternity/template:14]: Clean up comments

File: srcpkgs/eternity/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+hostmakedepends="cmake"
+makedepends="SDL2-devel SDL2_mixer-devel SDL2_net-devel zlib-devel"
+short_desc="Advanced Boom-compatible Doom engine with vanilla compatibility"
+maintainer="Antkibo <hxd.1996@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/team-eternity/eternity"
+distfiles="https://github.com/team-eternity/eternity/archive/${version}.tar.gz https://github.com/Wohlstand/libADLMIDI/archive/v1.5.0.1-1.tar.gz"
```
Comment (ericonr) [inline srcpkgs/eternity/template:21]: Split into two lines.

File: srcpkgs/eternity/template [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+checksum="bb9f69af38394dadd5b4d619b4ce8930728e1e61c70085379f53978be228c333
+ f19b18f35564f4db6f5d35d820655ea48e2cf8ffb193037fad74eecbabb35f07"
+
+post_extract() {
+	cp -r ${wrksrc}/libADLMIDI-1.5.0.1-1/* ${wrksrc}/${pkgname}-${version}/adlmidi
```
Comment (ericonr) [inline srcpkgs/eternity/template:26]: Should probably move the entire folder. `rmdir aldmidi` then `mv`.

**Review by ericonr — COMMENTED** [review]
Not sure how I feel about yet another DOOM.

File: srcpkgs/eternity/INSTALL.msg (hunk 1 above)
Comment (Antkibo) [reply srcpkgs/eternity/INSTALL.msg]: If one clones the repository those directories are there. This problem happens with the release tar.

**Comment by Antkibo** [comment]
> ut ye

File: srcpkgs/eternity/template (hunk 4 above)
Comment (Antkibo) [reply srcpkgs/eternity/template]: Thanks. I made the changes.

**Pull request closed without merging**